src/plugins/data/public/ui/filter_bar/filter_editor/_filter_editor.scss

```diff
@@ -1,3 +1,5 @@
 .globalFilterEditor__fieldInput {
-  max-width: $euiSize * 13;
+  @include euiBreakpoint('m', 'l', 'xl') {
+    max-width: $euiSize * 14;
```

Comment:
`13` -> `14`, I'd say `13` was adapted for 400px width. Change to `14`, because with `420px` there is a redundant spacing between inputs.

Reply:
Seems this `420` number is trying to be used multiple times. I'd just put all the styles in the SASS (removing the inline styles) and create local variable for reuse using this calculation.

Reply:
In other place it is part of EUI api: https://github.com/elastic/kibana/blob/master/src/plugins/data/public/ui/filter_bar/filter_item.tsx#L231,

With that in mind, do you think it is should be somehow forced to 420 with SASS instead?

Reply:
If you want them to continue to match in the future, I'd recommend setting up a variable they can all use. Otherwise, there's nothing pointing to them being connected so if someone tweaks just one of them, the layout will break. Just trying to future-proof it.

Reply:
Updated! 
I still have two variables: one in SASS and another in JavaScript. 
The reason for it is that I don't see a simple way to replicate this: https://github.com/elastic/kibana/pull/79448/commits/5e954bbd24dcdb16cf21b11dc0ba072b8b3b1ce2 with SASS.